www/src/App.tsx

```diff
@@ -1,21 +1,54 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+// Libraries
+import React, { FC } from 'react';
+import { Route, BrowserRouter as Router, Switch } from 'react-router-dom';
 
-function App(): JSX.Element {
+// Material UI
+import { Container, CssBaseline, ThemeProvider } from '@material-ui/core';
+
+// Components and Routes
+import { Header, Section } from './components/Header';
+import { Landing } from './routes/Landing';
+import { ResumeReview } from './routes/ResumeReview';
+import { MockInterview } from './routes/MockInterview';
+import { Community } from './routes/Community';
+
+// Styles
+import theme from './styles/theme';
+
+// Constants
+import { COMMUNITY, COMPE_PLUS, MOCK_INTERVIEW, RESUME_REVIEW } from './util/constants';
+
+const header_sections: Section[] = [
+    { title: RESUME_REVIEW, url: '/resume-review' },
+    { title: MOCK_INTERVIEW, url: '/mock-interview' },
+    { title: COMMUNITY, url: '/community' },
+];
+
+const App: FC = () => {
     return (
-        <div className="App">
-            <header className="App-header">
-                <img src={logo} className="App-logo" alt="logo" />
-                <p>
-                    Edit <code>src/App.tsx</code> and save to reload.
-                </p>
-                <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-                    Learn React
-                </a>
-            </header>
-        </div>
+        <ThemeProvider theme={theme}>
+            <CssBaseline />
```

Comment:
Shouldn't this be added to the head of the HTML doc?

Reply:
Are you referring to `ThemeProvider` or `CSSBaseline` ?  Regardless, I think they should / can just be in the `App` component: https://stackoverflow.com/questions/51658793/react-js-apply-material-ui-cssbaseline